src/plugins/ardura/Scrollscope/1.4.0.yaml

```diff
@@ -0,0 +1,25 @@
+---
+name: Scrollscope
+author: Ardura
+homepage: https://github.com/ardura/Scrollscope
+description: A simple scrolling oscilloscope in rust with multichannel inputs and an analyzer
+date: 2024-07-12T07:00:00.000Z
+license: GPL3
```

Comment:
Can you update to a value that matches GitHub licenses?
`license: gpl-3.0`

Reply:
updated in latest commit